gitinstall-hp.md

```diff
@@ -0,0 +1,98 @@
+# Installing git on HP-UX
+
+Topics in this document:
+
+* [Installing via Depot Packages](#installing-via-depot-packages)
+* [Installing via Internet](#installing-via-internet)
+  * [Caching Depot Packages](#caching-depot-packages)
+* [Installing via Source Code](#installing-via-source-code)
+* [Basic git functionality test](#basic-git-functionality-test)
+
+## Installing via Depot packages
+
+This is the recommended install method if you have read access to the wsscfs network share.
+
+* Login as root on the target HPv3 system : `ssh root@myHPv3system`
```

Comment:
You can probably assume the person knows how to log in as root! :smile: Okay to leave this, just chuckled a bit.